Accmpagnement speaker (#68)
philippart-s/blog: _posts/2023-01-01-accompagnement-speaker.md

---
title: "Toujours par deux ils vont. Ni plus, ni moins. Le maître et son apprenti."
excerpt: "De Yoda"
classes: wide
date: 2023-01-14
categories:
  - Articles
tags:
  - Humeur
---
![Yoda et QUI-GON]({{ site.url }}{{ site.baseurl }}/assets/images/accompagnement-speaker/yoda-qui-gon.png){: .align-center}
[www.starwars.com](https://www.starwars.com/databank/yoda?image_id=5390fdbd0a172d315d000484){:style="font-size: smaller"}{:target="_blank"}{: .align-right}

Depuis quelques temps, il m'arrive d'accompagner des personnes qui souhaitent se lancer dans l'aventure de devenir speaker dans les conférences techs.

> Du coup, je me suis dis, tient et si je vous partageais comment j'en suis arrivé là et ce que j'y ai gagné.

Reviewer: ```suggestion
Du coup, je me suis dit, tiens ! et si je vous partageais comment j'en suis arrivé là et ce que j'y ai gagné.
```

## Un peu de vocabulaire 🔠

![Professeur en classe]({{ site.url }}{{ site.baseurl }}/assets/images/accompagnement-speaker/teaching.jpg){: .align-center}
[Austrian National Library](https://unsplash.com/fr/photos/GxB6Pbi4Jzg?utm_source=unsplash&utm_medium=referral&utm_content=creditShareLink){:style="font-size: smaller"}{:target="_blank"}{: .align-right}

C'est assez amusant car le titre de l'article est justement ce que je n'aime pas dans ce type de démarche : une notion de sachant et de non sachant.

> Mais j'aimai bien le parallèle avec cette citation 😉 (et avouons le, je sui un fan de star wars !).  

Reviewer: ```suggestion
Mais j'aimais bien le parallèle avec cette citation 😉 (et avouons-le, je suis un fan de Star Wars !).  
```

En effet, pour moi je ne suis pas dans la position du professeur avec ses étudiants, ou de l’artisan avec l'apprenti.
Je ne suis pas là pour former la personne en lui distillant mon savoir mais plus pour l'aider à trouver sa voie dans ce nouveau défi qu'elle s'est lancé.
Je me vois aussi comme le public zéro ou le commité de CFP zéro, la version draft en quelque sorte 😅.

# Le pourquoi et le quand 🤔

![Fusée qui décolle]({{ site.url }}{{ site.baseurl }}/assets/images/accompagnement-speaker/start-nasa.jpg){: .align-center}
[NASA](https://unsplash.com/fr/photos/n463SoeSiVY?utm_source=unsplash&utm_medium=referral&utm_content=creditShareLink){:style="font-size: smaller"}{:target="_blank"}{: .align-right}

La démarche d'accompagner des personnes dans le désir d'être speaker n'a pas été une chose évidente, cela remonte à, à peu près, deux ans avant l'écriture de cet article.
Je n'étais (et je ne suis toujours pas) un spécialiste de la prise de parole en conférences.

> Mais le hasard fait, parfois, bien les choses 😉 : avec l'association que j'ai co-créé, [TADx](https://www.tadx.fr){:target="_blank"}, nous avons co-organisé un tremplin pour donner sa chance à tout un groupe de speakers de donner leur premier talk dans une conférence.  

Reviewer: ```suggestion
Mais le hasard fait, parfois, bien les choses 😉 : avec l'association que j'ai co-créée, [TADx](https://www.tadx.fr){:target="_blank"}, nous avons co-organisé un tremplin pour donner sa chance à tout un groupe de speakers de donner leur premier talk dans une conférence.  
```

Le tremplin était co-organisé avec [Le Camping des Speakers](https://camping-speakers.fr/){:target="_blank"}, la conférence pour laquelle les participantes et participants pouvaient gagner une place, et [Crafts Records](https://craftsrecords.org/){:target="_blank"}, une association parisienne qui offre la possibilité d'accompagner des speakers pour les aider à donner leur premier talk.

> Avec TADx on a proposé d'héberger cet event qui se déroulait comme suit : 

Reviewer: ```suggestion
Avec TADx on a proposé d'héberger cet event à Tours. Voici comment cela s'est déroulé : 
```

 - une phase d'accompagnement des speakers dans la phase d'écriture du CFP,
 - une phase d'accompagnement des speakers dans la phase de création du talk,
 - des répétitions,
 - l'accompagnement du jour J,

Lors de la réception de cet event on m'a proposé de faire partie de l'équipe d’accompagnants 😱.

> Mon premier réflexe a été de dire non, syndrome de l'imposteur ou juste connaissance de la réalité, je ne me sentais pas légitime à conseiller quelqu'un sur comment construire et donner un talk alors que moi même j'en avais donné 2 ou 3 de toute ma vie !

Reviewer: ```suggestion
Mon premier réflexe a été de dire non, syndrome de l'imposteur ou juste connaissance de la réalité, je ne me sentais pas légitime à conseiller quelqu'un sur comment construire et donner un talk alors que moi-même j'en avais donné 2 ou 3 de toute ma vie !
```

Mais voilà, on ne se refait pas, je me suis laissé embarquer 🤪.

C'est à partir de là que je ne me suis pas senti coach mais plus accompagnateur, oeil externe.
J'avais tout de même un avantage : cela faisait plus d'une douzaine d'années que j'écumais les conférences tech en tant que spectateur.
Et je pense, que rien que ça, cela aide à donner un feedback constructif.

> L'autre avantage est, qu'organiser (avec ma présidente) un meetup mensuel, me permet aussi d'avoir certains réflexes dans l'analyse d'un talk (est-ce qu'il va plaire, est-il assez large, ...).

Reviewer: ```suggestion
L'autre avantage est qu'organiser (avec ma présidente) un meetup mensuel, me permet aussi d'avoir certains réflexes dans l'analyse d'un talk (est-ce qu'il va plaire, est-il assez large, ...).
```

Et c'était donc parti, je me suis retrouvé à accompagner une collègue qui avait soumis, pour la première fois, un talk !

## Le comment 🤓

![Graine qui pousse]({{ site.url }}{{ site.baseurl }}/assets/images/accompagnement-speaker/growth.jpg){: .align-center}
[Jeremy Bishop](https://unsplash.com/fr/photos/vGjGvtSfys4?utm_source=unsplash&utm_medium=referral&utm_content=creditShareLink){:style="font-size: smaller"}{:target="_blank"}{: .align-right}

> Loins de moi l'idée de vous donner un cours sur comment accompagner de futur.e.s speakers mais plus sur comment, à titre personnel, je vois ça.  

Reviewer: ```suggestion
Loin de moi l'idée de vous donner un cours sur comment accompagner de futur.e.s speakers mais plus sur comment, à titre personnel, je vois ça.  
```

Comme je l'ai déjà indiqué, pas question de faire un cours et surtout ne pas de m’approprier le talk, c'est pour moi le piège principal à éviter lorsque l'on accompagne une personne.  
Exit donc les phrases du type : _à ta place ..._, _moi j'aurai plutôt ..._, _je connais cette techno ce n'est pas comme ça_, ...  

Mon objectif est de me mettre dans la peau d'un spectateur, comme en conférence : 
 - est-ce que je comprends le message ?
 - est-ce fidèle à l’abstract que j'ai lu ?
 - est-ce que je décroche pour une raison ou une autre ?
 - est-ce que, en tant que débutant, je comprends ce qui est raconté ?
 - les slides, démos sont-il clairs ?
 - ...

Bref le client zéro lambda de base 😊.

Une autre chose, qui paraît évidente : je laisse parler la personne le plus possible !

> Eh oui c'est sont talk, même si j'ai quelques idées je préfère toujours laisser la personne me présenter ses idées plutôt que l'orienter sur des terrains qu'elle n'aurait pas prévu d'aller.

Reviewer: ```suggestion
Eh oui c'est sont talk, même si j'ai quelques idées je préfère toujours laisser la personne me présenter ses idées plutôt que l'orienter sur des terrains où  elle n'aurait pas prévu d'aller.
```

Et dans les conditions réelles, slides, démo, timing, etc ...
C'est tout bête, mais si la personne a déjà une certaine appréhension de le faire face à moi alors que l'on se connaît, devant 20, 50 ou 300 personnes cela sera pire !
Et autre chose j'en profite pour tout noter en termes de timing, afin, notamment de donner des indications pour que la ou le speaker puisse savoir là ou elle / il passe trop temps, peut gagner du temps, ...

Il y a pleins d'autres "outils" disponibles, comme je l'ai dit je ne suis pas coach ou spécialiste en gestion humaine, donc créez vos propres routines ou outils.
A s'adapter aussi selon la personne que l'on accompagne.

Je vais terminer cette partie par ce qui est, selon moi, le plus important : accompagner la personne.
Et pour cela je vais prendre ma casquette de parent, eh oui, au final pour moi ce rôle d’accompagnant est le même que celui que l'on peut avoir avec nos enfants.
Tenir par la main, aider à progresser mais en faisant que la personne se construise elle-même en tant que speaker, sans le faire à sa place.
En la ou le relevant quant elle / il trébuche, en l’encourageant, en la / le rassurant, ...

> Honnêtement dans la plupart des accompagnements que j'ai pu faire, la production de la personne est déjà à 90% fini et de qualité et ne demande pas de changements.

Reviewer: ```suggestion
Honnêtement dans la plupart des accompagnements que j'ai pu faire, la production de la personne est déjà à 90% terminée et de qualité et ne demande pas de changements.
```

> Je suis là pour les 10%, souvent dû à quelques trucs et astuces méconnus lorsque l'on se lance dans ce genre d'aventure 🤗.

Reviewer: ```suggestion
Je suis là pour les 10%, souvent dus à quelques trucs et astuces méconnus lorsque l'on se lance dans ce genre d'aventure 🤗.
```

> Encore une fois, lorsque l'on accompagne une personne pour un talk, notre rôle principal, selon moi, est de la guider, provoquer le déclique.

Reviewer: ```suggestion
Encore une fois, lorsque l'on accompagne une personne pour un talk, notre rôle principal, selon moi, est de la guider, provoquer le déclic.
```

Et dernière chose, lorsque c'est possible, j'essaie d'être là lors du passage en conférence pour gérer l'avant et l'après avec la personne. 
Ces deux moments sont importants et c'est toujours réconfortant d'avoir du monde à ses côtés.

## Et moi dans tout ça ? 🤔

Il paraît que l'acte gratuit n'existe pas.  
Vous avez deux heures ... 🤯

![lettres qui affichent listen learn love]({{ site.url }}{{ site.baseurl }}/assets/images/accompagnement-speaker/listen-learn-love.jpg){: .align-center}
[Brett Jordan](https://unsplash.com/@brett_jordan){:style="font-size: smaller"}{:target="_blank"}{: .align-right}

> Bien sûr, je ne pense pas m'être lancé dans cette aventure parceque cela m'apportait quelque chose d'évident.

Reviewer: ```suggestion
Bien sûr, je ne pense pas m'être lancé dans cette aventure parce que cela m'apportait quelque chose d'évident.
```

> J'ai, simplement, découvert au fur et à mesure, que cet exercice m'apportait énormément de choses à titre personnel. 

Reviewer: ```suggestion
J'ai simplement découvert, au fur et à mesure, que cet exercice m'apportait énormément de choses à titre personnel. 
```

En effet, durant l'accompagnement vous devez vous remettre en question afin d'être sûr de promulguer les bons conseils.
Cela a comme effet immédiat de vous faire poser des questions sur vos propres talks, est-ce que j'applique vraiment tout ce que je suis en train de conseiller ?  

> Vous l'aurez compris rien de mieux que l'effet miroir pour se rendre compte qu'il est toujours plus facile de juger les autres que soit même 🧐.  

Reviewer: ```suggestion
Vous l'aurez compris rien de mieux que l'effet miroir pour se rendre compte qu'il est toujours plus facile de juger les autres que soi-même 🧐.  
```

Les émotions sont fortes, quasi plus fortes que lors de mes propres talks 😳.
Mais alors quelle fierté, je peux vous l'assurer !
Et pas de la fierté du genre "_elle ou il a bien écouté ce que je lui ai dit_" mais juste de la fierté (je me rends compte que c'est compliqué de faire passer l'émotion par l'écrit).

## En conclusion

Vous l'aurez compris, l'accompagnement d'un.e futur.e speaker est une aventure émotionnelle forte mais qui apporte beaucoup 🤩.
C'est une forme de partage comme on peut le retrouver dans l'open source par exemple mais basé plus sur l'humain.  
Enfin j'aimerai indiquer qu'il existe de plus en plus de conférences qui réservent des créneaux pour les primos speakers (comme [Devoxx France](https://www.devoxx.fr/){:target="_blank"} par exemple) voir, vont même jusqu'à proposer un accompagnement aux primos speakers dans leur CFP (comme ça a été le cas avec [Touraine Tech](https://touraine.tech/){:target="_blank"}).  
Et puis, comment ne pas citer les meetups, comme [TADx](https://www.tadx.fr/){:target="_blank"} par exemple, qui laissent très souvent leur chance aux primos speakers, voir leurs proposent un accompagnement 🤗

> Si vous êtes arrivé jusque là merci de m'avoir lu et si il y a des coquilles n'hésitez pas à me faire une [issue ou PR](https://github.com/philippart-s/blog){:target="_blank"} 😊.

Reviewer: ```suggestion
Si vous êtes arrivés jusque là merci de m'avoir lu et si il y a des coquilles n'hésitez pas à me faire une [issue ou PR](https://github.com/philippart-s/blog){:target="_blank"} 😊.
```
